### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by 719media**
php 7.0 in there twice. I know, I know, this is a super awesome pr

### CODE REVIEW

**Comment by derickr** [comment]
This is on purpose, the ``php: 7.0`` for this one is specifically for the running of the tests with ``SERVER_VERSION=3.0.15``.

**Comment by 719media** [comment]
But isn’t it already included for this version just a few lines up?

**Comment by 719media** [comment]
Oops! Thanks

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +20,6 @@ matrix:
     - php: 7.0
     - php: 7.1
     - php: 7.2
-    - php: 7.0
```
Comment (jmikola) [inline .travis.yml]: > But isn’t it already included for this version just a few lines up?

@719media: If I'm reading this diff correctly, it combines two separate jobs (PHP 7.2 with server 3.6.2 and PHP 7.0 with server 3.0.15) with one (PHP 7.2 with server 3.0.15).

For no real reason in particular, we run tests with all older server versions under PHP 7.0.

**Pull request closed without merging**